### PR TITLE
[ios] Update changelogs and bump podspec.

### PR DESCRIPTION
**Pull request by fabian-guerra**
podspec bump

### CODE REVIEW

File: platform/ios/CHANGELOG.md [hunk 1]
```diff
@@ -2,7 +2,39 @@
 
 Mapbox welcomes participation and contributions from everyone. Please read [CONTRIBUTING.md](../../CONTRIBUTING.md) to get started.
 
+## 4.0.3 - June 22, 2018
+
 * Fixed crash in `-[MGLStyle localizeLabelsIntoLocale:]` on iOS 9.3 (attempting to access a property that was introduced in iOS 10.0) ([#12123](https://github.com/mapbox/mapbox-gl-native/pull/12123))
```
Comment (julianrex) [inline platform/ios/CHANGELOG.md]: This needs adjusting to match the log under the 4.1.0 section

File: platform/macos/CHANGELOG.md [hunk 2]
```diff
@@ -1,8 +1,34 @@
 # Changelog for Mapbox Maps SDK for macOS
 
+ ## 0.7.2 - June 22, 2018
+
 * Fixed crash in `-[MGLStyle localizeLabelsIntoLocale:]` on iOS 9.3 (attempting to access a property that was introduced in iOS 10.0) ([#12123](https://github.com/mapbox/mapbox-gl-native/pull/12123))
```
Comment (julianrex) [inline platform/macos/CHANGELOG.md]: And here - should match the one under the 0.8.0 section

File: platform/ios/CHANGELOG.md (hunk 1 above)
Comment (fabian-guerra) [reply platform/ios/CHANGELOG.md]: According to @friedbunny 
> In any case: the changelog is chronological, so 4.0.3 should be placed before 4.1.0.

File: platform/macos/CHANGELOG.md (hunk 2 above)
Comment (fabian-guerra) [reply platform/macos/CHANGELOG.md]: same as above

File: platform/ios/CHANGELOG.md (hunk 1 above)
Comment (julianrex) [reply platform/ios/CHANGELOG.md]: I meant the `Fixed crash ...` copy 😄

**Pull request merged**